### ISSUE TITLE
Space after colon error

### ISSUE THREAD

**Issue by mi2oon** [sasstools/sass-lint#468]
Hi there,

I get the following error in with my sass lint. I am using gulp-sass-lint which hasn't updated the sass-lint package with all the latest fixes. However, I tried with the latest release and had the same problem.

```
node_modules\gulp-sass-lint\node_modules\sass-lint\lib\rules\space-after-colon.js:17
        if (next.is('space')) {
                ^
TypeError: Cannot read property 'is' of undefined 
```

This sort of is similar to the following error. https://github.com/sasstools/sass-lint/issues/89. Anyway, let me know if you need more info. :)

**Comment by DanPurdy** [comment]
Hi @mi2oon thanks for the report, would you mind narrowing down and posting a sample of the code that this fails on and also let us know whether this is with `.sass` or `.scss`.

Thanks

**Comment by mi2oon** [comment]
@DanPurdy Sure :)

The code is quite simple. It's actually a mistype. Lets say we have the code below.

```sass
.fail{ color: red: }
```
Notice that instead of semi colon at the end there is a colon which clearly is wrong. However, instead of giving an error it just breaks my watch process. The file type is `.scss`. I can make the a PR fixing it just like in #89, like so.

```js
if (next.is('space')) ----->  if (next && next.is('space')) 
```

**Comment by DanPurdy** [comment]
Yeah, guessing theres a few others like this that we've not caught yet.

A PR would be appreciated if you want to.

Thanks!

**Comment by mi2oon** [comment]
@DanPurdy hehe.. no problem. PR has been made ;)
